src/main/java/edu/unc/lib/boxc/migration/cdm/services/AspaceRefIdService.java

```diff
@@ -71,6 +73,10 @@ public void generateBlankAspaceRefIdMapping() throws IOException {
     public void generateAspaceRefIdMappingFromHookIdRefIdCsv() throws IOException {
         assertProjectStateValid();
 
+        if (!hasProjectContriAndDescriFields()) {
```

Comment:
We should add a test to verify that this works. Can probably use the "monograph" that we already have sitting around in the test resources.